src/components/security_manager/test/crypto_manager_impl_test.cc

```diff
@@ -52,7 +52,8 @@ namespace {
 const size_t kUpdatesBeforeHour = 24;
 const std::string kAllCiphers = "ALL";
 const std::string kCaCertPath = "";
-
+const uint32_t serviceNumber = 2u;
```

Comment:
@ValeriiMalkov 
- named wrongly according to SDL Coding Style

Reply:
@AByzhynar Fixed at the commit e60c4f93668e3be86a3d3f9ac5fe1cac3b3b6f78